diff --git a/lessons.txt b/lessons.txt
new file mode 100644
index 0000000..e69de29
--- /dev/null
+++ b/lessons.txt
diff --git a/login.py b/login.py
index 191f63e..7317df8 100644
--- a/login.py
+++ b/login.py
@@ -32,15 +32,9 @@
 	driver.close()
 	
 	extractHomework(a)
-except as e:
-	sendHomework(e)
-	print(e)
+	
 times = ['14:29', '14:30', '14:31', '14:32', '14:33', '14:34']
 date = datetime.now(pytz.timezone('Asia/Yekaterinburg'))
 for i in times:
     if date.strftime('%H:%M') == i:
-        sendHomework('.')
-'''
-if c == 'New': 
-	
-'''
\ No newline at end of file
+        sendHomework('.')
\ No newline at end of file
